### ISSUE TITLE
Use Frigate mask definitions

### ISSUE THREAD

**Issue by ragenhe** [jakowenko/double-take#79]
Great project!! Thank you for all the work.

It would be great if we could copy / paste motion masks from our frigate setup.

Or maybe there is something already similar.

My problem is Face Detection is running against my TV as a second person.

**Comment by jakowenko** [comment]
I pushed a new beta build to dynamically adjust the mask if a size is passed. I also slightly changed the format of the mask definitions. They look like this now.

```yaml
masks:
  living-room:
    size: 1920x1080
    mask:
      - 1864,649,1884,494,1687,473,1681,614
      - 590,467,766,283,1146,264,1176,805,809,834
    visible: true
```

**Comment by ragenhe** [comment]
This is working like a charm @jakowenko!!

**Comment by jakowenko** [comment]
Awesome! I'm glad to hear!!!

**Comment by jakowenko** [comment]
Closing as this was merged into release `1.0.0`
